config/resources/app2.yaml

```
@@ -1,3 +1,7 @@
+defaults:
+  - subscription_id: notreal-default-subscription
+    resource_group: notreal-default-rg
```

```suggestion
  subscription_id: notreal-default-subscription
  resource_group: notreal-default-rg
```
Don't think this needs to be an array/list/slince whatever it is :D